src/main/webapp/app/exam/participate/exam-cover/exam-participation-cover.component.ts

```diff
@@ -129,14 +129,7 @@ export class ExamParticipationCoverComponent implements OnInit, OnDestroy {
     startExam() {
         if (this.testRun) {
             this.examParticipationService.saveStudentExamToLocalStorage(this.exam.course!.id!, this.exam.id!, this.studentExam);
-            if (this.hasStarted()) {
-                this.onExamStarted.emit(this.studentExam);
-            } else {
-                this.waitingForExamStart = true;
-                this.interval = window.setInterval(() => {
-                    this.updateDisplayedTimes(this.studentExam);
-                }, 100);
-            }
+            this.onExamStarted.emit(this.studentExam);
```

Comment:
@FuchsDominik why did you change the implementation here? are you sure the logic is not necessary for a test run?

Reply:
During testing i discovered that to get into this branch, the exam run has to be a test run. (See line 130) hasStarted() automatically returns true if the exam is in a test run, that's why it is not possible that the check in line 132 return false. Therefore the else clause will never get executed. @anditurdiu confirmed my findings, right @anditurdiu ?

Reply:
Logic-wise, an instructor does not have to wait for the exam test run as the test run starts automatically

Reply:
ok sounds good